src/client/sky.cpp

```diff
@@ -573,9 +539,23 @@ void Sky::update(float time_of_day, float time_brightness,
 					(rangelim(m_time_brightness, 0.05, 0.15) - 0.05) * 10 * 0.625);
 			}
 
-			video::SColorf pointcolor_moon_f(0.5 * pointcolor_light,
-				0.6 * pointcolor_light, 0.8 * pointcolor_light, 1);
-			if (m_moon_tonemap) {
+			video::SColorf pointcolor_moon_f;
+			if (m_default_tint) {
+				pointcolor_moon_f = video::SColorf(
+					0.5 * pointcolor_light,
+					0.6 * pointcolor_light,
+					0.8 * pointcolor_light,
+					1
+				);
+			} else {
+				pointcolor_moon_f = video::SColorf(
+					m_sky_params.moon_tint.getRed() * pointcolor_light,
+					m_sky_params.moon_tint.getGreen() * pointcolor_light,
+					m_sky_params.moon_tint.getBlue() * pointcolor_light,
+					1
```

Comment:
The offending bug;

get[Red/Green/Blue] returns 0-255; not 0 - 1.0f.

The fix is to divide the results by 255, then multiply by `pointcolor_light`.